src/pages/landing/deriv-go/index.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+// import { graphql, useStaticQuery } from 'gatsby'
+import { Banner, DerivGoContent, GoFooterBanner, GoFooter } from './_lazy-load.js'
+// import Layout from 'components/layout/layout'
```

Comment:
We can remove the commented lines if not needed ya 🙏

Reply:
Amended in the latest commit